### PR TITLE
Add n18113

### PR DESCRIPTION
**Pull request by jnewbery**

### CODE REVIEW

File: _posts/2020-03-18-#18113.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  1677](https://github.com/bitcoin/bitcoin/pull/1677)) where it was keyed by
+  transaction. Pieter Wuille describes this change in a [recent Chaincode Podcast
+  episode](https://podcast.chaincode.com/2020/01/27/pieter-wuille-1.html).
+
+- In v0.15 the coins database was changed to be keyed by transaction output
+  ([PR 10195](https://github.com/bitcoin/bitcoin/pull/10195)). For more
+  information on that change see [this blog
+  post](https://johnnewbery.com/post/whats-new-in-bitcoin-core-v0.15-pt1/).
+
+- The 'prune' in ultraprune came from the fact that once all of the outputs
```
Comment (jonatack) [inline _posts/2020-03-18-#18113.md]: Afaict "ultraprune" isn't introduced in the study notes (aside from in the podcast) before being referred to here.

File: _posts/2020-03-18-#18113.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  UTXO set. We looked at the coins database in a [previous PR Review Club
+  (#17487)](./17487). The notes for that Review Club meeting include an
+  excellent description of the caching for the coins database.
+
+- The coins database is a key-value store. It was introduced in v0.8 ([PR
+  1677](https://github.com/bitcoin/bitcoin/pull/1677)) where it was keyed by
+  transaction. Pieter Wuille describes this change in a [recent Chaincode Podcast
+  episode](https://podcast.chaincode.com/2020/01/27/pieter-wuille-1.html).
+
+- In v0.15 the coins database was changed to be keyed by transaction output
```
Comment (jonatack) [inline _posts/2020-03-18-#18113.md]: perhaps set off "output" in italics to denote what changed

File: _posts/2020-03-18-#18113.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  ([PR 10195](https://github.com/bitcoin/bitcoin/pull/10195)). For more
+  information on that change see [this blog
+  post](https://johnnewbery.com/post/whats-new-in-bitcoin-core-v0.15-pt1/).
+
+- The 'prune' in ultraprune came from the fact that once all of the outputs
+  from a transaction had been spent, that entry could be removed or 'pruned'
+  from the coins database (note that this is a different concept from
+  a 'pruned node', which deletes old block files that it no longer needs).  The
```
Comment (jonatack) [inline _posts/2020-03-18-#18113.md]: pruned nodes might be a well enough known concept to not need quotes

File: _posts/2020-03-18-#18113.md (hunk 1 above)
Comment (jnewbery) [reply _posts/2020-03-18-#18113.md]: Good spot. I've updated the text above.

**Comment by jnewbery** [comment]
Thanks for the review @jonatack . I've taken all your suggestions and force-pushed

**Pull request merged**